hw/lint/tools/dvsim/common_lint_cfg.hjson

```diff
@@ -18,7 +18,7 @@
   build_log:  "{build_dir}/lint.log"
   // We rely on fusesoc to run lint for us
   build_cmd:  "fusesoc"
-  build_opts: ["--cores-root {proj_root}/hw",
+  build_opts: ["--cores-root {proj_root}",
```

Comment:
Nit: Should the first two commits in this PR be squashed for atomicity and this hunk moved to join them? (If you're not doing any other major surgery, it's probably not worth it, but I thought I should flag this up).

Reply:
Sure, I can squash them, no problem.